core/src/test/java/com/ritense/valtimo/service/CamundaProcessServiceIntTest.java

```diff
@@ -63,6 +66,21 @@ void shouldDeployNewProcess() throws IOException, ProcessNotUpdatableException {
         Assertions.assertTrue(definitions.stream().anyMatch(processDefinition -> processDefinition.getKey().equals("deployedProcess")));
     }
 
+    @Test
```

Comment:
Can you also write a test to verify a non-dmn non-bpmn file is handled correctly?